<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_home">
         <property name="text">
          <string>HOME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_tabelas">
         <property name="text">
          <string>TABELAS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_sobre">
         <property name="text">
          <string>SOBRE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_contatos">
         <property name="text">
          <string>CONTATOS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_outros">
         <property name="text">
          <string>OUTROS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="pg_mestre">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pg_tabelas">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="txt_file"/>
              </item>
              <item>
               <widget class="QPushButton" name="btn_abrir">
                <property name="text">
                 <string>ABRIR</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>ESTOQUE</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTreeWidget" name="tw_estoque">
                    <column>
                     <property name="text">
                      <string>NF-e</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Série</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Cliente</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Uf</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Municipio</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Cod_item</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Qtd</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Descrição</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Uni</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Espécie</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Valor</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Data_importação</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Usuário</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>SAÍDA</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTreeWidget" name="tw_saida">
                    <column>
                     <property name="text">
                      <string>NF-e</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>SÉRIE</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>DATA_IMPORTAÇÃO</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>DATA_SAÍDA</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>USUÁRIO</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QPushButton" name="btn_importar">
                   <property name="text">
                    <string>IMPORTAR</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_7">
                   <property name="text">
                    <string>GERAR SAÍDA</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_estorno">
                   <property name="text">
                    <string>ESTORNO</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_home">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#00503a;&quot;&gt;SISTEMA &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#00503a;&quot;&gt;DE GERENCIAMENTO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_usuario">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TELA DE CADASTRO</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CADASTRAR USUÁRIO</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>NOME      </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_nome"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>USUÁRIO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_usuario"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>SENHA    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_senha"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>SENHA2  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_senha2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>PERFIL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmb_perfil">
            <item>
             <property name="text">
              <string>ADMINISTRADOR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>USUÁRIO</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cad_usuario">
            <property name="text">
             <string>CADASTRAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cadastrar">
        <property name="text">
         <string>CADASTRAR USUARIO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
